src/nomos/agent/GenCodeCopyright

```diff
@@ -1,17 +1,5 @@
-***************************************************************
- Copyright (C) 2006-2009 Hewlett-Packard Development Company, L.P.
- 
- This program is free software; you can redistribute it and/or
- modify it under the terms of the GNU General Public License
- version 2 as published by the Free Software Foundation.
+/*
+ SPDX-FileCopyrightText: © 2006-2009 Hewlett-Packard Development Company, L.P.
 
- This program is distributed in the hope that it will be useful,
- but WITHOUT ANY WARRANTY; without even the implied warranty of
- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- GNU General Public License for more details.
-
- You should have received a copy of the GNU General Public License along
- with this program; if not, write to the Free Software Foundation, Inc.,
- 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
-
- ***************************************************************
+ SPDX-License-Identifier: GPL-2.0-only
+*/
```

Comment:
Please remove the `/*` and `*/` from the file content.

The content of the file gets copied to other files in format:
```c
/*
 * <Content of this file>
 */
```

The `*/` causes them to be:
```c
/*
 * /*
 SPDX-FileCopyrightText: © 2006-2009 Hewlett-Packard Development Company, L.P.

SPDX-License-Identifier: GPL-2.0-only
*/
 */
```

And it is causing code compilation failures.